elasticsearch/Dockerfile

```diff
@@ -20,4 +20,4 @@ ARG BUILD_NUMBER=-1
 LABEL io.astronomer.docker=true
 LABEL io.astronomer.docker.build.number=$BUILD_NUMBER
 
-RUN yum -y update bind-license
+RUN yum -y update bind-license dbus
```

Comment:
Patching a CVE? Should we separate this into a new PR for audit purposes?

Reply:
Yes please. Let's not overload PRs

Reply:
Please create a separate PR please

Reply:
done https://github.com/astronomer/ap-vendor/pull/40